tests/unit/test_utils.py

```diff
@@ -113,3 +122,44 @@ def test_absolute_path_not_exist(self) -> None:
         result = get_file_path(filename="~/xxxx/yyyy/doesnotexist.xyz")
 
         assert result is None
+
+
+@mark.parametrize(
+    "a,b,expected",
+    [
+        param(
+            {1: 2, 2: 5},
+            {1: 3},
+            {1: 3, 2: 5},
+            id="simple",
+        ),
+        param(
+            {1: {2: 3}, 2: 5},
+            {1: {2: 4}},
+            {1: {2: 4}, 2: 5},
+            id="nested",
+        ),
+        param(
+            {1: 2, 2: 5},
+            {1: {3: 4}},
+            raises(RuntimeError),
+            id="merge-dict-into-scalar",
+        ),
```

Comment:
how would you feel about moving this to a separate test? i think it would make the tests more readable and would also mean you don't have to import the `RaisesContext` from the private `_pytest` api

Reply:
Done in https://github.com/dbatten5/maison/pull/89/commits/681d47d62f1394f39c96d29df83c43fb80301410.

Reply:
Oh, whoops, I misunderstood you.

Reply:
Updated in https://github.com/dbatten5/maison/pull/89/commits/72d00b3c7d07b8d916e6e778a15d22875a587df1.